### PR TITLE
refactor the network port setting for AWS

### PR DESCRIPTION
**Pull request by NeoZhangJianyu**
## Description

refactor the description of network port setting for AWS.

## Issues

the description of network port setting is not clear.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

**Comment by kevinintel** [comment]
please update readme of gaudi and ChatQnA

File: ChatQnA/docker_compose/intel/cpu/xeon/README.md [hunk 1]
```diff
@@ -45,11 +45,10 @@ It will automatically download the docker image on `docker hub`:
 ```bash
 docker pull opea/chatqna:latest
 docker pull opea/chatqna-ui:latest
+docker pull opea/nginx:latest
 ```
 
-If you want to build docker by yourself, please refer to 'Build Docker Images' in below.
-
-> Note: The optional docker image **opea/chatqna-without-rerank:latest** has not been published yet, users need to build this docker image from source.
+To use the latest or special version, you could build docker image from source by yourself, please refer to 'Build Docker Images' in below.
```
[the remark this answers is not in the document]
Comment (NeoZhangJianyu) [reply ChatQnA/docker_compose/intel/cpu/xeon/README.md]: It's existed. so no need to high light it.

**Comment by NeoZhangJianyu** [comment]
> please update readme of gaudi and ChatQnA

yes, update them.

File: ChatQnA/README.md [hunk 2]
```diff
@@ -69,9 +72,15 @@ docker pull opea/chatqna:latest
 docker pull opea/chatqna-ui:latest
 ```
 
-If you want to build docker by yourself, please refer to `built from source`: [Guide](docker_compose/intel/cpu/xeon/README.md).
+#### Build Docker Image from Source
+
+In following cases, you could build docker image from source by yourself.
+
+- Download the docker image fault.
```
Comment (xiguiw) [inline ChatQnA/README.md:79]: `fault` default?

Comment (xiguiw) [inline ChatQnA/README.md:79]: `Download the docker image fault.`?

File: ChatQnA/README.md [hunk 3]
```diff
@@ -52,7 +52,10 @@ To set up environment variables for deploying ChatQnA services, follow these ste
 
 ### Quick Start: 2.Run Docker Compose
 
+#### Download Docker Image on `docker hub`
```
Comment (kevinintel) [inline ChatQnA/README.md:55]: please remove

File: ChatQnA/docker_compose/intel/cpu/xeon/README.md [hunk 4]
```diff
@@ -36,6 +36,8 @@ To set up environment variables for deploying ChatQnA services, follow these ste
 
 ## Quick Start: 2.Run Docker Compose
 
+### Download Docker Image on `docker hub`
```
Comment (kevinintel) [inline ChatQnA/docker_compose/intel/cpu/xeon/README.md:39]: please remove

File: ChatQnA/docker_compose/intel/hpu/gaudi/README.md [hunk 5]
```diff
@@ -37,6 +37,8 @@ To set up environment variables for deploying ChatQnA services, follow these ste
 
 ## Quick Start: 2.Run Docker Compose
 
+### Download Docker Image on `docker hub`
```
Comment (kevinintel) [inline ChatQnA/docker_compose/intel/hpu/gaudi/README.md:40]: please remove

File: ChatQnA/docker_compose/nvidia/gpu/README.md [hunk 6]
```diff
@@ -37,6 +37,8 @@ To set up environment variables for deploying ChatQnA services, follow these ste
 
 ## Quick Start: 2.Run Docker Compose
 
+### Download Docker Image on `docker hub`
```
Comment (kevinintel) [inline ChatQnA/docker_compose/nvidia/gpu/README.md:40]: please remove

File: ChatQnA/docker_compose/intel/cpu/xeon/README.md [hunk 7]
```diff
@@ -344,6 +325,53 @@ docker compose -f compose_vllm.yaml up -d
 
 ### Validate Microservices
 
+Note, before verify the microservices by curl or API, please make sure the ports of the microservices are opened in the firewall of the cloud node.
+
+> See one example below. Please open up these ports in the EC2 instance based on the IP addresses you want to allow:
+
+```
+redis-vector-db
```
Comment (yinghu5) [inline ChatQnA/docker_compose/intel/cpu/xeon/README.md:333]: remove the long list,  just  refer to ## Network Port & Security

File: ChatQnA/docker_compose/intel/cpu/xeon/README.md [hunk 8]
```diff
@@ -344,6 +325,53 @@ docker compose -f compose_vllm.yaml up -d
 
 ### Validate Microservices
 
+Note, before verify the microservices by curl or API, please make sure the ports of the microservices are opened in the firewall of the cloud node.
```
Comment (yinghu5) [inline ChatQnA/docker_compose/intel/cpu/xeon/README.md:328]: Note, before verify the microservices by curl or API, please make sure the ports like 5173/6007 are accessible.  To open the ports, please refer to  ## Network Port & Security

File: ChatQnA/docker_compose/intel/cpu/xeon/README.md (hunk 7 above)
Comment (NeoZhangJianyu) [reply ChatQnA/docker_compose/intel/cpu/xeon/README.md:333]: ## Network Port & Security refer to this chapter too.
Here is the port list for microservices.
It's very useful to user when open ports.

File: ChatQnA/README.md (hunk 2 above)
Comment (NeoZhangJianyu) [reply ChatQnA/README.md:79]: yes

**Review by yinghu5 — APPROVED** [review]
LGTM

File: ChatQnA/docker_compose/intel/cpu/xeon/README.md [hunk 9]
```diff
@@ -344,6 +321,8 @@ docker compose -f compose_vllm.yaml up -d
 
 ### Validate Microservices
 
+Note, before verify the microservices by curl or API, please make sure the **ports** of the microservices are opened in the firewall of the cloud node.
```
Comment (lvliang-intel) [inline ChatQnA/docker_compose/intel/cpu/xeon/README.md:324]: please add a limitation "if access from remote client"

Comment (yinghu5) [inline ChatQnA/docker_compose/intel/cpu/xeon/README.md]: Note, before verify the microservices by curl or API, please make sure the **ports** of the microservices are opened in the firewall of the cloud node if access from remote client.

**Review by lvliang-intel — APPROVED** [review]

**Comment by mkbhanda** [comment]
Thank @NeoZhangJianyu and team!

**Pull request merged**